code/modules/events/melting_spawning.dm

```diff
@@ -0,0 +1,43 @@
+/datum/round_event_control/melting
+	name = "Spawn Melting"
+	typepath = /datum/round_event/ghost_role/nightmare
+	max_occurrences = 1
+	min_players = 40
+	earliest_start = 1 HOURS //it's jungle fever basically
```

Comment:
why does it start to spawn at 1 hour and no earlier, like 40 mins in? how rare is it?

Reply:
//it's jungle fever basically

Reply:
if people don't see it enough, i'll bump it up.

Reply:
>//it's jungle fever basically

I still don't know what is meant by this